echopype/convert/set_groups_azfp.py

```diff
@@ -9,6 +9,18 @@
 from ..utils.coding import set_time_encodings
 from .set_groups_base import SetGroupsBase
 
+PHASE_PARAMS = ["burst_interval", "pings_per_burst", "average_burst_pings"]
+
+PHASE_FREQ_PARAMS = [
```

Comment:
These two lists are used only in `set_vendor`. Having them here, with a global, module-wide scope, seems a bit misleading. Plus, I think it'd be more helpful for readability if they were in fact declared at the start of `set_vendor`, because that's where they'll be used.